bin/wp-env-config.sh

```diff
@@ -18,5 +18,12 @@ wp rewrite flush
 wp core version --extra
 wp plugin list
 wp theme activate storefront
+wp plugin is-installed Basic-Auth
+if [ $? -eq 1 ]
+then
+  wp plugin install https://github.com/WP-API/Basic-Auth/archive/master.zip --activate
+else 
+  wp plugin activate Basic-Auth
+fi
```

Comment:
After approving this PR before, I noticed the following section. This plugin should already be installed and activated by default due to the following entry:

https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/c8a5041817eb672a8490453731bd3dccc5dfa58a/.wp-env.json#L3-L7

Therefore, I'd say the following section is not necessary, isn't it?